config/manager/kustomization.yaml

```diff
@@ -11,4 +11,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: ghcr.io/solacedev/pubsubplus-eventbroker-operator
-  newTag: v1.0.0-rc.3
+  newTag: 1.0.2-dev
```

Comment:
what is this new tag?

Reply:
basically, the docker image to be used, is autogenerated based on the image